sql/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectExecutionManager.scala

```diff
@@ -118,11 +116,6 @@ private[connect] class SparkConnectExecutionManager() extends Logging {
 
     sessionHolder.addExecuteHolder(executeHolder)
 
-    executionsLock.synchronized {
-      if (!executions.isEmpty()) {
-        lastExecutionTimeMs = None
-      }
-    }
```

Comment:
nice; it doesn't have to be set to None, because listActiveExecutions may directly depend on the list being empty

Reply:
Exactly :-)